custom/git-app-deployer/main.go

```diff
@@ -177,22 +182,13 @@ func DeleteApplication(appVars *AppVars, delay int, clientset *kubernetes.Client
 
 	log.Infof("[Status]: FilePath for App Deployer is %v", appVars.filePath)
 	log.Info("[Status]: Revert application has been started")
-	if err := DeleteSockShop("/var/run/"+appVars.filePath, appVars.namespace); err != nil {
+	if err := DeleteApp("/var/run/"+appVars.filePath, appVars.namespace); err != nil {
 		return err
 	}
 
 	if err := CheckPodStatusForRevert(appVars.namespace, appVars.timeout, 2, clientset); err != nil {
 		return err
 	}
-	log.Info("[Status]: Application pods are in terminating state")
-	if err := DeleteNamespace(clientset, appVars.namespace); err != nil {
-		if k8serrors.IsNotFound(err) {
-			log.Infof("[Status]: %v Namespace not exist", appVars.namespace)
-		} else {
-			return err
-		}
-	}
-	log.Info("[Status]: Namespace deleted successfully")
```

Comment:
why we removed this section?

Reply:
Now we are using target namespace for application installation. We we delete namespace then entire setup will ruin